README.md

```diff
@@ -1 +1,56 @@
-# solution-pack-malware-response-using-symantec-solutions
+# Malware Response Using Symantec Solutions
+
+## Overview
+### Introduction
+This article describes the  Malware Response Using Symantec Solutions Pack. The solution pack demonstrates the scenarios and use cases around Malware resported by Symantec EDR.
+
+The scenario demonstrates and generates a demo alert for the Alert Type 'Malwarel' reported by Symantec EDR.
+
+The usecase deals with containment and investigation procedures once the Symantec EDR reports a Malware Alert.
+
+    Step 1: Get asset details from Symantec EDR
+    Step 2: Get File Reputation using Symantec DeepSight Intelligence
+    Step 3: If File Reputation is Malicious then ask Analyst for Endpoint Isolation 
```

Comment:
File Reputation --> why first letter is in caps?

Reply:
done